src/api/app/views/webui2/shared/_editor.html.haml

```diff
@@ -9,14 +9,14 @@
     .sticky-top.bg-light{ id: "top_#{uid}" }
       .card-header.d-flex.justify-content-between
         .input-group.input-group-sm
-          %button.btn.btn-danger.btn-sm.disabled.save{ id: "save_#{uid}" }
+          %button.btn.btn-danger.btn-sm.save{ id: "save_#{uid}", disabled: true }
```

Comment:
why do you need to use disabled as prop instead of as class? :thinking:

Reply:
Because that's how buttons are getting [disabled](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/Attribute/disabled):smile: